config/gateway/gateway-plugin/gateway-plugin.yaml

```diff
@@ -31,6 +36,24 @@ spec:
       labels:
         app: gateway-plugins
     spec:
+      affinity:
+        podAntiAffinity: # pods are placed on different nodes
+           requiredDuringSchedulingIgnoredDuringExecution:
+           - labelSelector:
+               matchExpressions:
+                 - key: app
+                   operator: In
+                   values:
+                     - gateway-plugins
+             topologyKey: "kubernetes.io/hostname"
+        nodeAffinity: # prevent gateway pod to be placed on gpu node.
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: vke.node.gpu.schedule
```

Comment:
this is not common labels, should only exist in VKE.

Reply:
Yes. I have added here such that it does not get missed on using default deployment, and also provided same config in release overlay with a comment to change label value per their env.

Open to suggestion for naming.

Reply:
@varungup90 I think it's better for default ones to be directly used in public clouds, vke specific labels can be moved to our own overlay.  WDYT?